src/desktop/apps/ssr-experiments/App.tsx

```diff
@@ -1,6 +1,10 @@
 import React from "react"
-import { BorderBox } from "@artsy/palette"
+import { BorderBox, Box } from "@artsy/palette"
 
 export const App = () => {
-  return <BorderBox>hello</BorderBox>
+  return (
```

Comment:
What is this file used for?

Reply:
This is my little test bed for testing SSR / experimental stuff. We still have some non-breaking issues around rehydration; was trying to solve them in here.